packages/inventory-compliance/src/Utilities/FilterBuilderConfigBuilder.js

```diff
@@ -43,13 +43,13 @@ const BASE_FILTER_CONFIGURATION = [
 ];
 
 const PASS_FILTER_CONFIG = {
-    type: conditionalFilterType.radio,
+    type: conditionalFilterType.checkbox,
     label: 'Passed',
     items: [
         { label: 'Show passed rules', value: 'passed' },
         { label: 'Hide passed rules', value: 'failed' }
```

Comment:
Slight discrepancy with the mocks here:

```suggestion
        { label: 'Passed rules', value: 'passed' },
        { label: 'Failed rules', value: 'failed' }
```